pages/profile/content/index.js

```diff
@@ -22,5 +22,17 @@ module.exports = merge({}, profileContent, {
   },
   notifications: {
     success: 'Changes saved!'
+  },
+  pil: {
+    training: {
+      title: 'Training',
+      certificate: {
+        number: 'Certificate number',
+        awarded: 'Date awarded',
+        body: 'Accreditation body',
+        modules: 'Modules completed'
+      },
+      none: 'No training added'
+    }
```

Comment:
I assume this content lives in asl/pages - could we import from there rather than repeating?

Reply:
done.